<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OT Master</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="2,1,2,1,6">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="4" rowspan="3">
       <widget class="QFrame" name="info_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Hours Remaining:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0" colspan="2">
            <widget class="QFrame" name="group_frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="allotted_group_ot">
                <property name="maximum">
                 <double>999999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Allotted Hours:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Allotted OT:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Group:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="group_name_val">
                <property name="text">
                 <string>No group selected</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="allotted_group_hours">
                <property name="maximum">
                 <double>999999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Used Hours:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Remaining Hours:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="normal_group_used_hours">
                <property name="text">
                 <string>0.0 Normal</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="ot_group_used_hours">
                <property name="text">
                 <string>0.0 Overtime</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="9" column="1">
          <widget class="QCheckBox" name="same_as_group_checkbox">
           <property name="text">
            <string>Same as group</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="person_name_val">
           <property name="text">
            <string>No employee selected</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Overtime Remaining:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="hours_used_val">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="employee_remaining_ot">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="allotted_person_ot"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="name_label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="employee_remaining_hours">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QPushButton" name="add_timeclock_btn">
           <property name="text">
            <string>Add Timeclock</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Allotted Overtime:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="allotted_person_hours"/>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="hours_used_combo_box">
           <item>
            <property name="text">
             <string>Total hours used:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hours used for today:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hours used for this week:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hours used for this month:</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Allotted Hours:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="add_group_btn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="name_input">
        <property name="placeholderText">
         <string>Employee Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QListWidget" name="group_list"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="group_input">
        <property name="placeholderText">
         <string>Group Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="add_person_btn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QListWidget" name="name_list"/>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLineEdit" name="search_edit">
        <property name="placeholderText">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Database"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Database">
   <property name="text">
    <string>Import Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
